src/parser_utils.js

```diff
@@ -14,17 +14,38 @@ function childrenByName(node, name) {
 }
 
 /**
- * Parses a node text (for legacy support).
- * @param  {Object} node - The node to parse the text from.
+ * Parses a node value
+ * @param  {Object} node - The node to parse the value from.
  * @return {String}
  */
-function parseNodeText(node) {
+function parseNodeValue(node) {
+  const childNodes = node && node.childNodes && [...node.childNodes];
+
+  // Trying to find and parse cdata as JSON
+  const cdatas =
+    childNodes &&
+    childNodes.filter(childNode => childNode.nodeName === '#cdata-section');
+  if (cdatas && cdatas.length > 0) {
+    try {
+      return JSON.parse(cdatas[0].data);
+    } catch (e) {}
+  }
+
+  // Didn't find any cdata or failed to parse it
```

Comment:
So for what I understood we prioritize `CDATA` content over `TEXT`. Fine for me, but we should update the function JSdoc to explain that.